docs/README.md

```diff
@@ -0,0 +1,98 @@
+# JoliGAN Documentation
+
++ [Datasets](#datasets)
++ [Dataloaders](#dataloaders)
++ [Training](#training)
++ [Inference](#inference)
+
++ [Models and Options](options.md)
++ [Project Source Overview](overview.md)
+
+## Datasets
+
+### CycleGAN Datasets
+
+To train a model on your own datasets, you need to create a data folder with two subdirectories `trainA` and `trainB` that contain images from domain A and B. You can test your model on your training set by setting `--phase train` in `test.py`. You can also create subdirectories `testA` and `testB` if you have test data.
+
+You should **not** expect our method to work on just any random combination of input and output datasets (e.g. `cats<->keyboards`). From our experiments, we find it works better if two datasets share similar visual content. For example, `landscape painting<->landscape photographs` works much better than `portrait painting <-> landscape photographs`. `zebras<->horses` achieves compelling results while `cats<->dogs` completely fails.
```

Comment:
Warning: this is copy/paste from cyclegan readme, it does not truely apply to joligan. We'll write a section with our own observations later.

Reply:
True, I will remove this note